pandas/_libs/hashtable_func_helper.pxi.in

```diff
@@ -30,7 +34,7 @@ dtypes = [('float64', 'float64', 'float64_t'),
 cdef build_count_table_{{dtype}}(ndarray[{{dtype}}] values,
                                  kh_{{ttype}}_t *table, bint dropna):
 {{else}}
-cdef build_count_table_{{dtype}}({{dtype}}_t[:] values,
+cdef build_count_table_{{dtype}}(const {{dtype}}_t[:] values,
```

Comment:
we can/should do this independent of the complex dtypes right?

Reply:
True. I saw this inconsistency, because due to https://github.com/cython/cython/issues/2251 my second attempt didn't work for const memory views, but worked for this two functions.